LithoUtils/Classes/Base/Models/CodableUILabelStyle.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CodableUILabelStyle.swift
+//  LithoUtils
+//
+//  Created by Remmington Damper on 1/12/22.
+//
+
+import Foundation
+import UIKit
+import Prelude
+import LithoOperators
+
+open class CodableUILabelStyle : CodableViewStyle {
+    open var labelText: String?
```

Comment:
Let's get rid of this one – text isn't so much style related as data related, so no need to include it in this class